src/base/porting.h

```diff
@@ -43,13 +43,15 @@
 #endif
 
 #if defined (WINDOWS)
-#define IMPORT_VAR 	__declspec(dllimport)
-#define EXPORT_VAR 	__declspec(dllexport)
-#include <WinBase.h>
+#ifdef _EXPORTING
```

Comment:
This seem like to be modified at CBRD-21754.
please check it.

Reply:
I'm going to merge CBRD-21754 first.